hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -729,6 +729,34 @@ final Stream<DynamicTest> claimTokenAirdropSenderAccountPays() {
                 getAccountBalance(RECEIVER).hasTokenBalance(FUNGIBLE_TOKEN, 1)));
     }
 
+    @HapiTest
+    @DisplayName("both associated and not associated FTs and receiver sig required")
+    final Stream<DynamicTest> withTwoTokensAndReceiverSigReq() {
+        final String ALICE = "ALICE";
+        final String BOB = "BOB";
+        return hapiTest(flattened(
+                cryptoCreate(ALICE).balance(ONE_HUNDRED_HBARS),
+                cryptoCreate(BOB)
+                        .balance(ONE_HUNDRED_HBARS)
+                        .maxAutomaticTokenAssociations(0)
+                        .receiverSigRequired(true),
+                createFT(FUNGIBLE_TOKEN_1, ALICE, 1000L),
+                createFT(FUNGIBLE_TOKEN_2, ALICE, 1000L),
+                tokenAssociate(BOB, FUNGIBLE_TOKEN_1),
+                tokenAirdrop(
+                                moving(10, FUNGIBLE_TOKEN_1).between(ALICE, BOB),
+                                moving(10, FUNGIBLE_TOKEN_2).between(ALICE, BOB))
+                        .payingWith(ALICE),
+                getAccountBalance(BOB).hasTokenBalance(FUNGIBLE_TOKEN_1, 0),
```

Comment:
If Bob is already associated, why would the balance not be transferred immediately ?

Reply:
Because he has receiverSigRequired and hasn't signed the transaction.

Reply:
ahh yes .. makes sense. Thank you!